From 5e514db80998d0015f1f0c3ad83fce7c71cc1ba6 Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@lanl.gov>
Date: Fri, 17 Mar 2023 09:06:27 -0600
Subject: [PATCH] version set manually

---
 src/jobflow/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/jobflow/_version.py b/src/jobflow/_version.py
index 592426c..cf8282c 100644
--- a/src/jobflow/_version.py
+++ b/src/jobflow/_version.py
@@ -4,4 +4,4 @@
     __version__ = get_distribution("jobflow").version
 except DistributionNotFound:
     # package is not installed
-    __version__ = ""
+    __version__ = "0.1.10"
