From 36ff3a1a85e07d1010f72843d9107fa5f99795a7 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 17 Mar 2023 11:50:39 -0600
Subject: [PATCH] Update _version.py

---
 src/jobflow/_version.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/jobflow/_version.py b/src/jobflow/_version.py
index 592426c..d56029e 100644
--- a/src/jobflow/_version.py
+++ b/src/jobflow/_version.py
@@ -1,7 +1,7 @@
 from pkg_resources import DistributionNotFound, get_distribution
 
 try:
-    __version__ = get_distribution("jobflow").version
+    __version__ = "0.1.10"
 except DistributionNotFound:
     # package is not installed
     __version__ = ""
